src/components/Sum/Sum.jsx

```diff
@@ -1 +1,7 @@
-// export const Sum = () => ();
+export const Sum = ({ a = 0, b = 0 }) => (
+  <p>
+    { `Sum of ${a} and ${b} is ${a + b}` }
```

Comment:
This is unnecessary.
```suggestion
    {`Sum of ${a} and ${b} is ${a + b}`}
```